src/StateWorkflowServiceProvider.php

```diff
@@ -49,22 +49,40 @@ protected function configPath()
         return __DIR__.'/../config/workflow.php';
     }
 
+    /**
+     * Return migrations path.
+     *
+     * @return string
+     */
+    private function migrationPath()
+    {
+        return __DIR__.'/../database/migrations';
+    }
+
     /**
      * Publish config file.
      */
     protected function publishConfig()
     {
-        $this->publishes([
-            $this->configPath() => config_path('workflow.php'),
-        ], 'config');
+        if ($this->app->runningInConsole()) {
```

Comment:
Why do this?

Reply:
publishing of assets should only happen via artisan command. its newer approach Laravel is using

Reply:
The service provider should only use this if you ask for the files to be published with `vendor:publish`. Wondering why this is being done.